README.md

```diff
@@ -1,2 +1,13 @@
 # Adapter
 this repo contains Adapter design pattern implementation using C#
+
+## UML
+
+  <img src="https://raw.githubusercontent.com/NorahMAlmutairi/Adapter/main/Adapter.png"/>
```

Comment:
The relationship between the adapter and the compound should be "implements" since compound should be an interface.